shoppingfluxexport.php

```diff
@@ -597,6 +597,50 @@ public function generateFeed()
                 echo '<from/>';
                 echo '<to/>';
             }
+            
+            $specificPrices = SpecificPrice::getIdsByProductId($product->id);
```

Comment:
Why not put this in the if (Tools::getValue('discount') == 1) condition to avoid unecessary database requests ?